prometheus-to-sd/main.go

```diff
@@ -143,7 +144,7 @@ func readAndPushDataToStackdriver(stackdriverService *v3.Service, gceConf *confi
 		}
 		metrics, err := translator.GetPrometheusMetrics(sourceConfig.Host, sourceConfig.Port)
 		if err != nil {
-			glog.Warningf("Error while getting Prometheus metrics %v", err)
+			glog.Warningf("Error while getting Prometheus metrics %v for component %v", err, sourceConfig.Component)
```

Comment:
I'm OK with setting -v to 2 on master, but on the user's nodes this should not always be in logs